1.12.2-/gradle.properties

```diff
@@ -1,4 +1,12 @@
-org.gradle.jvmargs=-Xmx6G
+org.gradle.jvmargs=-Xmx3G
 
-LoaderVersion = 0.14.19
-LoomVersion = 1.1-SNAPSHOT
\ No newline at end of file
+#Fabric Settings
+LoaderVersion=0.14.21
+LoomVersion=1.1-SNAPSHOT
+
+#Preprocessor Settings
+DiscombobulatorVersion=1.2.1-SNAPSHOT
+
+#Jar Settings
+ArchiveName=DiscombobulatorTemplate
+ModVersion=1.0.0
```

Comment:
I recommend this for naming:
```yaml
# fabric
loader_version=0.14.21
loom_version=1.1-SNAPSHOT

# preprocessor
discombobulator_version=1.2.1-SNAPSHOT

# mod
mod_name=DiscombobulatorTemplate
mod_version=1.0.0
```

Reply:
Great, time to rename every damn thing in the build.gradle